tests/conftest.py

```diff
@@ -2067,6 +2067,17 @@ def compare_running_config(pre_running_config, cur_running_config):
             return False
 
 
+@pytest.fixture(scope="function", autouse=True)
+def skip_traffic_test(request):
```

Comment:
Also need to improve the fixture name to make it more specific and explicit. How about `skip_traffic_test_on_kvm`?

Reply:
Will we have other platform needs to only skip traffic test? If so, conditional yaml is easy to extend, if not, we can use skip_traffic_test_on_kvm

Reply:
Agree, fixed in new commit